ca_gazebo/launch/create_empty_world.launch

```diff
@@ -1,5 +1,5 @@
 <launch>
-	<arg name="env" default="empty"/>
+	<arg name="env" default="playground"/>
```

Comment:
Why did you edit this?

Reply:
I edited this because there's no empty.pgm map, maybe you could help me find it elsewhere?

Reply:
Lucas is working on it. Maybe you could talk to him to speed it up 😛

Reply:
Rebase against `kinetic-devel` and you should have the changes.